proposals/lip-0028.md

```diff
@@ -3,15 +3,15 @@ LIP: 0028
 Title: Define schema and use generic serialization for transactions
 Author: Iker Alustiza <iker@lightcurve.io>
 Discussions-To: https://research.lisk.io/t/define-schema-and-use-generic-serialization-for-transactions/211
-Type: Informational
+Type: Standards Track
 Created: 2020-02-18
 Updated: 2020-07-21
 Requires: 0027
 ```
 
 ## Abstract
 
-This LIP defines a generic serialization for transactions by specifying the appropriate JSON schema. The proposed serialization method is applied in two steps, first to the asset property, where type-specific properties are defined, and then to the base transaction object. It will be used for signing but also for storing and transmitting transactions in the Lisk protocol. Further, this shall be the default serialization method for custom transactions created via the Lisk SDK.
+This LIP defines a generic serialization for transactions by specifying the appropriate JSON schema. The proposed serialization method is applied in two steps, first to the asset property, where type-specific properties are defined, and then to the transaction object. It will be used for signing but also for storing and transmitting transactions in the Lisk protocol. Further, this shall be the default serialization method for custom transactions created via the Lisk SDK.
```

Comment:
Consider the following suggestion for the second sentence. I tried to avoid writing "type-specific" as there is no more transaction type. Moreover, "transaction base object" may be more clear as the transaction object is rather the whole object including the nested asset.

> The proposed serialization method is applied in two steps, first to the transaction asset, which contains the custom properties specific to the transaction, and then to the transaction base object, which is the same for all transactions.

Reply:
changed, thanks